index.ts

```diff
@@ -0,0 +1,17 @@
+import { Client } from "./src/client";
+import { Account } from "./src/services/account";
+import { Avatars } from "./src/services/avatars";
+import { Database } from "./src/services/database";
+import { Locale } from "./src/services/locale";
+import { Storage } from "./src/services/storage";
+import { Teams } from "./src/services/teams";
+
+export {
+    Client,
+    Account,
+    Avatars,
+    Database,
+    Locale,
+    Storage,
+    Teams,
+};
```

Comment:
How does this sdk is going to be used? Would consumers necessarily use all of its services all the time?

If not, this way of exporting is non-tree-shakeable and you should consider exporting this services by themselves, you can do it as:
```
export { Client } from "./src/client";
```